ledger/src/account/repo.rs

```diff
@@ -6,6 +6,7 @@ use uuid::Uuid;
 use super::entity::*;
 use crate::{error::*, primitives::*};
 
+/// Provides methods to interact with `Account` entities.
```

Comment:
This here is actually their repository (as you can tell by the file name). `Repository which implements persistence for the Account entity` or something like that perhaps.